requirements.txt

```diff
@@ -1,26 +1,29 @@
-Django==2.2.19
-wagtail==2.11.2
+Django==3.1.7
+wagtail==2.12.3
 # Pin treebeard to <4.5, see: https://github.com/wagtail/wagtail/issues/6820
 django-treebeard<4.5
```

Comment:
This doc saids that django-treebeard can be unpinned at the version of 

wagtail 2.12.3 

https://docs.wagtail.io/en/stable/releases/2.12.3.html
https://github.com/wagtail/wagtail/commit/de88832899e7a78037776305a271b0d4b2d26ea5

I think the current version will work, and you might wanna try

Reply:
Thanks brian, good catch, will test that